chroma-manager/requirements.dev

```diff
@@ -18,4 +18,4 @@ pyflakes
 pygraphviz
 sphinx==1.1.3
 werkzeug
-iml-common
+iml-common>=1.0.5
```

Comment:
> we need to pin here because the unit tests don't clear packages inside the virtual environment used

But pinning isn't clearing is it?  I'm still not sure I understand what the problem is.

Doesn't `pip` always ensure the latest version is installed, upgrading an existing venv to the latest if it's not already?  Don't we always just want to use the latest version?

> if you want to resolve that it would be much appreciated

What problem(s) are resolved by clearing the venv and recreating it each time?  Maybe we need to get a ticket opened on this issue so that we can understand what the problem is and come up with a solution to whatever the problem is.